client/src/error.rs

```diff
@@ -88,6 +90,9 @@ impl fmt::Display for Error {
             Error::InvalidCookieFile => write!(f, "invalid cookie file"),
             Error::UnexpectedStructure => write!(f, "the JSON result had an unexpected structure"),
             Error::ReturnedError(ref s) => write!(f, "the daemon returned an error string: {}", s),
+            Error::Unsupported => {
```

Comment:
Could you change this to `UnsupportedArgument` and add a couple `&'static str` param describing (a) the RPC being called, (b) the argument that isn't supported.

Reply:
Sure, done